src/main/resources/application-custom.yaml

```diff
@@ -15,7 +15,7 @@ spring:
     check-location: false
     baselineOnMigrate: true
   datasource:
-    url: ${datasource.url}
+    url: jdbc:mysql://${DB_HOST}:${DB_PORT}/${DB_NAME}?${DB_PARAMS}
     username: ${datasource.username}
```

Comment:
Please also change these to:
```
DB_USER
DB_PASSWORD
```

Reply:
Should we just remove those completely and simply use SPRING_DATASOURCE_USERNAME and SPRING_DATASOURCE_PASSWORD ?

Reply:
No, I'd like to be consistent with the other `DB_` environment variable usage.

Reply:
Done